iocBoot/iocBPMRFFEApp/run.sh

```diff
@@ -14,6 +14,8 @@ device_prefix_var=CRATE_${CRATE_NUMBER}_BPM_${BPM_NUMBER}_PV_DEVICE_PREFIX
 export EPICS_PV_AREA_PREFIX=${!area_prefix_var}
 export EPICS_PV_DEVICE_PREFIX=${!device_prefix_var}
 
+export R_BPM=${EPICS_PV_DEVICE_PREFIX%RFFE}
+
```

Comment:
`EPICS_PV_DEVICE_PREFIX` is already what you want. We append RFFE in the definition of `R`.

We can rename it to `EPICS_PV_BPM_PREFIX` or something shorter to make this clearer.

Reply:
Nice catch, thanks. It hadn't worked correctly when I used `R`, but I didn't check where we added `RFFE`.